src/Symfony/Component/Routing/RouteCollectionBuilder.php

```diff
@@ -14,11 +14,16 @@
 use Symfony\Component\Config\Exception\LoaderLoadException;
 use Symfony\Component\Config\Loader\LoaderInterface;
 use Symfony\Component\Config\Resource\ResourceInterface;
+use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
+
+@trigger_error(sprintf('The "%s" class is deprecated since Symfony 5.1, use "%s" instead.', RouteCollectionBuilder::class, RoutingConfigurator::class), E_USER_DEPRECATED);
```

Comment:
@nicolas-grekas , this deprecation is always triggered in the kernel, because we cannot avoid using this class for BC. That's why the tests started failing: https://travis-ci.org/symfony/symfony/jobs/616003485#L4396. How can I fix that?

Reply:
Mark it as expected?

Reply:
Oh, let's remove this line then.